docs/application/dotnet/guides/tizen.circularUI/CircleDateTimeSelector.md

```diff
@@ -0,0 +1,97 @@
+
+In the Date type, you can change value of the item by touching the item and rotating bezel to set value of the year: month: day.
+For the Time type, you can change value of the hour: minute: AM / PM.
+
+ |![Date](data/CircleDateTimeSelector_DatePicker.png) | ![Time](data/CircleDateTimeSelector_TimePicker.png)|
+ |:--------------------------------------------------:|:--------------------------------------------------:|
+ |                      Date                          |                           Time                     |
+
+**WARNING: [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView), [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper) must be contained by `CirclePage` or [CircleSurfaceEffectBehavior](xref:Tizen.Wearable.CircularUI.doc.CircleSurfaceEffectBehavior) should be added in [Behaviors](https://developer.xamarin.com/api/type/Xamarin.Forms.Behavior/) of [Page](https://developer.xamarin.com/api/type/Xamarin.Forms.Page/) that contains these controls. If other `pages` contain these controls, it may cause exception or can't display control.**
+
+## Adding CircleDateTimeSelector at CirclePage
+
+You can set `CircleDateTimeSelector` at [CirclePage.Content](xref:Tizen.Wearable.CircularUI.doc.CirclePage). If you want to know how to add [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage), please refer to [CirclePage guide](https://samsung.github.io/Tizen.CircularUI/guide/CirclePage.html#create-circlepage).
+The following code shows [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage) with `CircleDateTimeSelector`.
+`RotaryFocusTargetName` attribute sets the currently focused control that is handled by rotating and display the focused control's circle object.
+If you don't set this property, control can't receive the [Rotary Event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary).
+
+`CircleDateTimeSelector` has the following properties:
+
+- ValueType : Time can be changed if it is Time and if it is a Date, the date can be changed.
+- DateTime : Sets or gets date/time.
+- MaximumDate : Sets or gets maximum date.
+- MimimumDate : Sets or gets minimum date.
+
+The example below sets the time picker by putting the `ValueType` value "Time". And set the current time to the `DateTime` property.
+The current time will be displayed on the screen when it is executed. Since the hour item is shown as having focus, the hour is changed when the bezel is turned. If you touch the minute item and turn the bezel, you can change the minute.
```

Comment:
When the code is executed, the current time will be displayed on the screen.

Since the focussed area on the screen shows the hour item, the hour item is changed when the bezel is turned.

To change the minutes, touch the minute item and turn the bezel.